stepwise-regression/script.whizzml

```diff
@@ -0,0 +1,99 @@
+;; A simple function to get the max value in a list
+(define (get-max xs) (reduce (lambda (x y) (if (> x y) x y)) (head xs) xs))
+
+;; Get feature names given ids
+(define (feature-names dataset-id ids)
+  (let (fields ((fetch dataset-id) "fields"))
+    (map (lambda (id) (fields [id "name"])) ids)))
+
+;; Create a dataset sample
+(define (sample-dataset ds-id rate oob)
+  (create-and-wait-dataset {"sample_rate" rate
+                            "origin_dataset"  ds-id
+                            "out_of_bag" oob
+                            "seed" "whizzml-example"}))
+
+;; Split a dataset into training and test sets
+(define (split-dataset ds-id rate)
+  (list (sample-dataset ds-id rate false)
+        (sample-dataset ds-id rate true)))
```

Comment:
instead of `sample-dataset` and `split-dataset` you can just use the standard library function `create-dataset-split`... in your case the call would be: `(create-dataset-split ds-id rate "whizzml-example")`